plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestPinotQueryBase.java

```diff
@@ -34,10 +36,13 @@
     protected final PinotConfig pinotConfig = new PinotConfig().setControllerUrls("localhost:9000");
 
     protected final PinotClient mockClusterInfoFetcher = new MockPinotClient(pinotConfig, getTestingMetadata());
+    protected final PinotToTrinoTypeConverter typeConverter = new PinotToTrinoTypeConverter(new TestingTypeManager());
```

Comment:
Replace  `TestingTypeManager` with `TESTING_TYPE_MANAGER`. Also, it would be nice to define a constant `PinotToTrinoTypeConverter TESTING_TYPE_CONVERTER` in MetadataUtil or somewhere and reuse it.

Reply:
Since TestingTypeManager is only used by the type converter I just instantiated it in the constructor. lmk if you want it extracted to a static final constant.